diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter2/KdocCommentsWarnTest.kt

```diff
@@ -401,7 +402,7 @@ class KdocCommentsWarnTest : LintTestBase(::KdocComments) {
                     |   ) {
                     |}
                 """.trimMargin(),
-            LintError(5, 4, ruleId, "${KDOC_NO_CONSTRUCTOR_PROPERTY.warnText()} /**...", false)
+            LintError(5, 4, ruleId, "${KDOC_NOT_CONSTRUCTOR_PROPERTY.warnText()} /**...", false)
```

Comment:
Tag of this test still @Tag(WarningNames.KDOC_**NO**_CONSTRUCTOR_PROPERTY)?

Btw, I think, that is such case, 
`KDOC_NO_CONSTRUCTOR_PROPERTY` == `all properties from the primary constructor should be documented in a @property tag in KDoc` 
is more appropriate, since this field **IS** property from the primary constructor, while
 `KDOC_NOT_CONSTRUCTOR_PROPERTY` == `only properties from the primary constructor should be documented in a @property tag in class KDoc`,
is more relevant to the situation, when user add documentation in a `@property` tag for non primary constructor's fields. Isn't it?

Reply:
I also think, that it's better to change the name of `KDOC_NOT_CONSTRUCTOR_PROPERTY ` it's very similar to `KDOC_NO_CONSTRUCTOR_PROPERTY `, but they are responsible for different things

Reply:
Yeah, that's a good catch. Fixed.